### PR TITLE
chore(site): add CreateTokenPage story

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
FYI: Tried to run a few scenarios but it was failing for me when trying to spy on the API methods. Opened a bug on Storybook repo: https://github.com/storybookjs/storybook/issues/26386

Related to https://github.com/coder/coder/issues/12263

### CODE REVIEW

File: site/src/pages/CreateTokenPage/CreateTokenForm.tsx [hunk 1]
```diff
@@ -21,6 +22,8 @@ import {
   customLifetimeDay,
 } from "./utils";
 
+dayjs.extend(utc);
```
Comment (aslilac) [inline site/src/pages/CreateTokenPage/CreateTokenForm.tsx:25]: I've seen Storybook catch so many of these "depending on a dayjs plugin that we don't actually install locally" bugs. 🐛

**Comment by aslilac** [comment]
Could you just specify `parameters.queries` on the individual stories instead of spying/mocking?

**Comment by BrunoQuaresma** [comment]
> Could you just specify `parameters.queries` on the individual stories instead of spying/mocking?

They don't work for mutations 😥

**Review by aslilac — APPROVED** [review]
ah, right. fair enough!

**Pull request merged**